app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -8,6 +8,8 @@ module ManageIQ::Providers::Lenovo::ManagerMixin
     nil       => 'basic_auth'
   }.freeze
 
+  USER_AGENT = "CFME/#{MiqServer.last&.version}".freeze
```

Comment:
Do not use the string "CFME" as part of the user agent string.  CFME is a downstream Red Hat thing only, and upstream you should use ManageIQ.  You can get the correct string by using `I18n.t("product.name")`.  That being said, you probably should not access the i18n catalog at the class-level context, so I recommend putting this into a method instead.  On that note, this seems like something that might be useful for *every* provider, so I would be OK with a class method in the core repo to get the user agent string.

Additionally, don't use MiqServer.last as you are technically then getting a random server object.  Instead, if you are going for the version, prefer `Vmdb::Appliance::VERSION` or if you want the build number, `Vmdb::Appliance::BUILD`.

Reply:
@Fryguy and @juliancheal  thanks for help me, I will work on this.